cloud/migrate-to-cloud/hypershift.md

```diff
@@ -218,6 +218,118 @@ To see all available configuration options for Hypershift, use this command:
 docker run -ti timescale/hypershift:0.4 --help
 ```
 
+### Filtering tables and schemas
```

Comment:
As per https://github.com/timescale/hypershift/issues/183, we will have `Hypershift` dedicated in the sidebar. So, I wonder if we should keep the explanation regarding filtering in a separate file?

Reply:
I've discussed with Vineeth that he will restructure the docs when the sidebar is added.

Reply:
Thanks for letting me know about that docs org issue. We can't move Hypershift in the docs right now, as we are going through a restructure. However, we already have it down to be reviewed after the restructure goes live. When that happens, we'll determine where it ends up in the left nav. Right now, though, it will remain under `migration`.